model/DeliveryExecutionGetterInterface.php

```diff
@@ -28,9 +31,6 @@ interface DeliveryExecutionGetterInterface
 
     /**
      * Retrieves delivery execution by id (or returns null if doesn't exist) accessible by passed $ltiProvider
```

Comment:
It seems we do not need this comment anymore, since we have typehint + returntype.